robot_smach_states/examples/navigation/example_force_drive.py

```diff
@@ -0,0 +1,36 @@
+# ROS
+import rospy
+import sys
+
+# TU/e Robotics
+from robot_skills.get_robot import get_robot_from_argv
+
+# Robot Smach States
+from robot_smach_states import ForceDrive
+
+
+if __name__ == "__main__":
+    assert len(sys.argv) == 2, "Please provide the robot name" \
+                               "e.g., 'python example_something.py hero'"
+
+    rospy.init_node("example_force_drive")
+
+    robot = get_robot_from_argv(index=1)
+
+    rospy.loginfo("Creating forward force drive state")
```

Comment:
Maybe print that the robot is going to move and ask for confirmation (`raw_input`). You don't want people to have the robot move into something when trying this.

Reply:
Good point, robot will now tell it is going to execute a force drive and waits for confirmation.